README.md

```diff
@@ -74,3 +74,42 @@ To update CKEditor in liferay-portal:
 3. Re-deploy the module with `gradlew clean deploy`.
 
 An example of this can be seen in [this](https://github.com/liferay/liferay-portal/commit/5b2ae3732d96f7f0dec6d35cb4de99f9d389c248) commit (look at the [`package.json`](https://github.com/liferay/liferay-portal/blob/5b2ae3732d96f7f0dec6d35cb4de99f9d389c248/modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json) file)
+
+## Patching
+
+1. Make sure you're update to date with the [superproject](https://github.com/liferay/liferay-ckeditor) repository:
+
+	```sh
+	git pull origin master
+	```
+
+2. Make sure the `ckeditor-dev` submodule is clean and up to date.
+
+	```sh
+  sh ./scripts/update.sh
+  ```
+
+3. Work on your changes:
+
+	This could be `cherry-pick`ing a previously created commit or manually editing a file, so this can't be automated.
+
+4. Create your commit, add your changes and write a good commit message.
+
+5. Navigate back to the superproject's root directory and create the patch:
+
+	```sh
+	cd ..
+	sh ./scripts/patch.sh
+	```
+
+7. Create a build of CKEditor containing the patches:
+
+	From the root of the superproject's directory, run
+
+	```sh
+	sh ./scripts/build.sh
+	```
+
+8. Don't forget to add the changes and commit
+
+
```

Comment:
Something is off with the syntax here because GitHub is failing to format it:

![liferay-ckeditor_README_md_at_adb1de7012376546b20a5d927296ba95a4d5ab6d_·_liferay_liferay-ckeditor](https://user-images.githubusercontent.com/7074/57457404-c6937680-726f-11e9-9127-763b38e040d9.png)

Reply:
Yes, I'll fix that thanks for the catch